cmd/podman/common/create_opts.go

```diff
@@ -246,27 +246,30 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig, cgroup
 		}
 	}
 
+	parts := strings.SplitN(string(cc.HostConfig.NetworkMode), ":", 2)
```

Comment:
nit: Why is this line so far away from where the parts variable is finally used?

Reply:
fixed, I just copied it from another place.